exatomic/base.py

```diff
@@ -79,3 +77,27 @@ def display_side_by_side(*args):
     html_str = ''.join([df.to_html() for df in args])
     display_html(html_str.replace('table','table style=\"display:inline\"'),
                  raw=True)
+
+def sym2isomass(symbol):
+    """
+    Function to get a mapper dictionary to get isotopic masses rather than
+    isotopically weigthed atomic masses.
+
+    Args:
+        symbol (list or iterable): Elements of interest
+
+    Returns:
+        masses (dict): Dictionary that can be used inplace of sym2mass
+
+    """
+    # remove duplicates
+    symbol = list(dict.fromkeys(symbol))
+    # get a dataframe made up of the given symbols
+    tmp = isotopedf.groupby('symbol').filter(lambda x: x['symbol'].unique() in symbol)
+    # sort it by the abundance and get the first entry in each element
+    df = tmp.sort_values(by=['af'], ascending=False).groupby('symbol').head(1)
+    # convert to a dict
+    df.index = df['symbol']
+    masses = df['mass'].to_dict()
```

Comment:
This might be better `df.set_index('symbol')['mass'].to_dict()`. There might be some `pandas` things going on behind the scenes here, which may be causing the test errors in travis?